index.html

```diff
@@ -0,0 +1,95 @@
+                    </svg>    
+                </div>
+            </div>
+        </header>
+        <section class="headline">
+            <div class="text-container">
+                <section class="headline-inner">
+                    <h1 class="primary-text">Evans Kupour</h1>
+                    <h2 class="supporting-text">Software Developer</h2>
+                </section>
+                <p>
+                    I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <hr class="hr-1">
+
+                <div class="support">
+                    <div class="frame">
+                        <div class="language">
+                            <ul>
+                                <li class="common">Language</li>
+                                <li>Javascript</li>
+                                <li>Ruby</li>
+                                <li>Html</li>
+                                <li>Css</li>
+                            </ul>
+                        </div>
+                        <div class="framework">
+                            <ul>
+                                <li class="common">Framework</li>
+                                <li>Bootstrap</li>
+                                <li>Ruby on Rails</li>
+                                <li>RSPec</li>
+                                <li>CapyBara</li>
+                                <li>Selenium</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="skills">
+                        <ul>
+                            <li class="common">Skills</li>
+                            <li>Codekit</li>
+                            <li>Gitlab</li>
+                            <li>Codepen</li>
+                            <li>Github</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
+                <hr>
+                <div class="social-media">
+                    <div class="my-text">
+                        <a href=""><i class="fa fa-twitter"></i></a>
+                        <a href=""><i class="fa fa-github"></i></a>
+                        <a href=""><i class="fa fa-facebook-f"></i></a>
+                        <a href=""><i class="fa fa-linkedin"></i></a>
+                        <a href=""><i class="fa fa-youtube"></i></a>
+                    </div>
```

Comment:
### Your Icons
![your](https://user-images.githubusercontent.com/50721479/185496149-7b38bac8-0f9a-435b-9555-8fa04a10d328.png)
### Figma Icons
![figma](https://user-images.githubusercontent.com/50721479/185496210-01aa11db-28f2-44f9-8423-8b268a477c5f.png)

- [ ] Please increase the size of your icons to match it with the Figma template :+1: as suggested by previous reviewer :blush: